ClrMD.Extensions/ClrMDSession.cs

```diff
@@ -123,7 +117,7 @@ public static ClrMDSession AttachToProcess(Process p, uint millisecondsTimeout =
 
             Detach();
 
-            DataTarget target = DataTarget.AttachToProcess(p.Id, millisecondsTimeout, attachFlag);
+            DataTarget target = DataTarget.AttachToProcess(p.Id, true);
```

Comment:
What is this boolean param here? Is that a replacement for AttachFlag that would be useful to expose?